ccmlib/scylla_repository.py

```diff
@@ -567,3 +562,37 @@ def run_scylla_unified_install_script(install_dir, target_dir, package_version):
     run('''{0}/install.sh --prefix {1} --nonroot{2}'''.format(
         install_dir, target_dir, install_opt), cwd=install_dir)
     run(f'''ln -s {install_dir}/scylla/conf conf''', cwd=target_dir)
+
+
+Architecture = Literal['x86_64', 'aarch64']
+
+
+def get_manager_latest_reloc_url(branch: str = "master", architecture: Architecture = None) -> str:
+    """
+    get the latest manager relocatable version of a specific branch
+    """
+    architecture = architecture or os.environ.get('SCYLLA_ARCH', 'x86_64')
+
+    url = f"https://s3.amazonaws.com/downloads.scylladb.com/manager/relocatable/unstable/{branch}/"
+    # filter only specific architecture
+    all_packages = [tar for tar in aws_bucket_ls(url) if architecture in tar]
+
+    # return latest
+    return f'https://s3.amazonaws.com/downloads.scylladb.com/{all_packages[-1]}'
+
+
+def get_manager_release_url(version: str = '', architecture: Architecture = None) -> str:
```

Comment:
we can also consolidate these two functions into a single one, like
```py
def get_manager_url(version: Optional[str] = None,
                    architecture: Architecture = None) -> str:
    base_url = 'https://s3.amazonaws.com/downloads.scylladb.com'
url = f'{base_url}/manager/relocatable/unstable/{branch}/'
    all_packages = reversed(aws_bucket_ls(url))
    if architecture is None:
        architecture = os.environ.get('SCYLLA_ARCH', 'x86_64')
    matched_packages = filter(lambda tar: architecture in tar, all_packages)

    if version is not None:
        version = version.replace('.', '\.')
        version_regex = re.compile(f'scylla-manager_{version}.*-0')
        matched_packages = (tar for tar in matched_packages if version_regex.search(tar))

    latest_package = next(matched_packages)
    return f'{base_url}/{latest_package}'
```

Reply:
the url isn't the same in both, the release are in one place, and the development version are if a different location.
I think it's o.k. to have two different function